.circleci/config.yml

```diff
@@ -349,50 +356,38 @@ workflows:
           requires:
             - checkout
 
-  react-next:
+  additional-tests:
     when:
-      equal: [react-next, << pipeline.parameters.workflow >>]
-    # triggers:
-    #   - schedule:
-    #       cron: '0 0 * * *'
-    #       filters:
-    #         branches:
-    #           only:
-    #             - master
+      and:
+        - equal: [additional, << pipeline.parameters.workflow >>]
+        - or:
+            - equal: [true, << pipeline.parameters.with-material-ui-6 >>]
+            - not:
+                equal: ['stable', << pipeline.parameters.with-react-version >>]
```

Comment:
With this condition, it won't be possible to run additional tests with material v6 only.
Is it necessary?

Reply:
yes, I see what you mean, nice catch.
before it was not equal `''`, but I had to add default.
I will update this

Reply:
It is `with-material-ui-6` is `true` or `with-react-version` not `stable`
So you can run ui-6 with any react version and you can run any non-stable version of react with v5 (stable with v5 is in the main pipeline)